server/src/test/java/org/opensearch/gateway/remote/RemoteClusterStateServiceTests.java

```diff
@@ -2057,12 +3121,6 @@ private void mockBlobContainerForGlobalMetadata(
                 .stream()
                 .collect(Collectors.toMap(Map.Entry::getKey, entry -> getFileNameFromPath(entry.getValue().getUploadedFilename())));
 
-            // ChecksumBlobStoreFormat<Metadata.Custom> customMetadataFormat = new ChecksumBlobStoreFormat<>(
```

Comment:
Nit: remove unused code

Reply:
I am removing the commented out code here